config.py

```diff
@@ -21,7 +24,12 @@ class Config(object):
     GOOGLE_TAG_MANAGER = os.getenv('GOOGLE_TAG_MANAGER', None)
     NON_DEFAULT_VARIABLES = ['SECRET_KEY', 'SECURITY_USER_NAME', 'SECURITY_USER_PASSWORD', 'JWT_SECRET']
     AVAILABILITY_BANNER = os.getenv('AVAILABILITY_BANNER', False)
-    
+
+    # Zipkin
+    ZIPKIN_DISABLE = bool(strtobool(os.getenv("ZIPKIN_DISABLE", "False")))
```

Comment:
Doesn't look like it's being used anywhere :/ unless there's some magic going on in which case...

We don't want it disabled by default? Does the docker-dev and ras-deloy configs need changing to disable this?

Reply:
This is being used in the Zipkin library https://github.com/qiajigou/flask-zipkin/blob/master/flask_zipkin.py#L68

Reply:
Disabling it will remove the zipkin headers from requests as well as sending messages to zipkin